client/src/performance/performance.jsx

```diff
@@ -0,0 +1,150 @@
+    this.handleEditToggle = this.handleEditToggle.bind(this);
+    this.handleNameChange = this.handleNameChange.bind(this);
+    this.handleOpenAtChange = this.handleOpenAtChange.bind(this);
+    this.handlePerformDateChange = this.handlePerformDateChange.bind(this);
+  }
+
+  componentDidReceiveProps(props) {
+    this.setState(getStateFromProps(props));
+  }
+
+  handleCloseAtChange(closeAt) {
+    this.setState({ closeAt });
+  }
+
+  handleConfirm() {
+    const { state } = this;
+
+    delete state.editing;
+    delete state.requesting;
+
+    this.props.onEdit({
+      ...state,
+      closeAt: state.closeAt.format(),
+      openAt: state.openAt.format(),
+      performDate: state.performDate.format()
+    });
+  }
+
+  handleEditToggle() {
+    this.setState(({ editing }) => ({ editing: !editing }));
+  }
+
+  handleNameChange({ target }) {
+    this.setState({ [target.name]: target.value });
+  }
+
+  handleOpenAtChange(openAt) {
+    this.setState({ openAt });
+  }
+
+  handlePerformDateChange(performDate) {
+    this.setState({ performDate });
+  }
+
+  render() {
+    const { editing, requesting } = this.state;
+    const { closeAt, name, openAt, performDate } = this.state;
+    const listExported = false;
+
+    return (
+      <Card>
+        <CardHeader
+          title={editing ?
+            <div className="Performance-dateWrapper">
+              <label>Performance Name:&nbsp;</label>
+              <input onChange={this.handleNameChange} name="name" value={name} />
+            </div> :
+            name
+          }
+          subtitle={editing ?
+            <div className="Performance-dateWrapper">
+              <label>Performance Date:&nbsp;</label>
+              <Datetime
+                onChange={this.handlePerformDateChange}
+                textFormat={performDateFormat}
+                value={performDate}
+              />
+            </div> :
+            `Performance on ${moment(performDate).format(performDateFormat)}`
+          }
+        />
+        {editing ?
+          <CardText>
+            <div className="Performance-dateWrapper">
+              <label>Opens at:&nbsp;</label>
+              <Datetime
+                onChange={this.handleOpenAtChange}
+                textFormat={openCloseDateFormat}
+                value={openAt}
+              />
+            </div>
+            <div className="Performance-dateWrapper">
+              <label>Closes at:&nbsp;</label>
+              <Datetime
+                onChange={this.handleCloseAtChange}
+                textFormat={openCloseDateFormat}
+                value={closeAt}
+              />
+            </div>
+          </CardText> :
+          <CardText>
+            <div>Opens at: {moment(openAt).format(openCloseDateFormat)}</div>
+            <div>Closes at: {moment(closeAt).format(openCloseDateFormat)}</div>
+          </CardText>
+        }
+        <CardActions>
+          {!listExported && (editing ?
```

Comment:
Remove this `listExported` check. Name and performance date can still be edited, regardless of whether or not it was exported already